scripts/gis-material-update/validate-deploy/modules/hsl.py

```diff
@@ -0,0 +1,48 @@
+import logging
+
+from modules.config import Config
+from modules.common import *
+from modules.gis_validate_deploy import GisProcessor
+
+class HslBuses(GisProcessor):
```

Comment:
The only differences between e.g. this file and `liikennevaylat.py` are the class name, the docstring under it and the module name. Having several copies of the same code makes maintenance harder since any change has to be applied to every copy of the code or they will start diverging.

Is it possible to add the common code to the abstract base class?